TestVectorsAwsCryptographicMaterialProviders/runtimes/python/test/internaldafny/extern/GetTestDirectoryExtern.py

```diff
@@ -0,0 +1,8 @@
+import TestWrappedMaterialProvidersMain
+import os
+import _dafny
+
+def GetTestVectorExecutionDirectory():
+    return _dafny.Seq(os.getcwd() + "/../../")
```

Comment:
If we were to publish the python test vectors how would this behave? Is it always the case that the execrution dir has to be two dirs up?

Reply:
This is only used in the Dafny tests.
We only really run these via `make test_python`, which enforces the execution directory as `runtimes/python`, which means the `/../../` is valid for any tests run from `make test_python`.
(This is also true for Java/NET tests, those also have an implicit expected root directory of `""`)

Reply:
that's class thank you!